backend/main.py

```diff
@@ -83,6 +87,13 @@ def resource_not_found_exception_handler(
     return JSONResponse(status_code=404, content={"message": str(e)})
 
 
+@app.exception_handler(EventRegistrationException)
+def event_registration_exception_handler(
+    request: Request, e: EventRegistrationException
+):
+    return JSONResponse(status_code=403, content={"message": str(e)})
+
+
```

Comment:
There's actually a more idiomatic way for feature specific exception handling to be registered! Let's keep `main.py` exceptions to those that are top-level of project.

First, add a file like this to the feature: https://github.com/unc-csxl/csxl.unc.edu/blob/main/backend/api/coworking/exception_handlers.py

Then, register the feature specific exception handler here: https://github.com/unc-csxl/csxl.unc.edu/blob/main/backend/main.py#L86-L89